types/ahocorasick/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts",
+        "ahocorasick-tests.ts"
```

Comment:
Add browser test that uses this as global. https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module